telemetry_core/src/main/java/com/newrelic/telemetry/SenderConfiguration.java

```diff
@@ -67,36 +69,77 @@ public static SenderConfigurationBuilder builder(String defaultUrl, String baseP
     private boolean auditLoggingEnabled = false;
     private String secondaryUserAgent;
 
+    /** Create a new Builder, with the default URL and base path. */
     public SenderConfigurationBuilder(String defaultUrl, String basePath) {
       this.defaultUrl = defaultUrl;
       this.basePath = basePath;
     }
 
+    /**
+     * Configure the New Relic Insert API key to use.
+     *
+     * @return this builder;
+     */
     public SenderConfigurationBuilder apiKey(String apiKey) {
       this.apiKey = apiKey;
       return this;
     }
 
+    /**
+     * Configure the {@link HttpPoster} implementation to use.
+     *
+     * @return this builder
+     */
     public SenderConfigurationBuilder httpPoster(HttpPoster httpPoster) {
       this.httpPoster = httpPoster;
       return this;
     }
 
+    /**
+     * Configure the *full* endpoint URL for data to be sent to, including the path.
```

Comment:
```suggestion
     * Configure the <strong>full</string> endpoint URL for data to be sent to, including the path.
```

I think Javadoc is html, not markdown, right?

Reply:
Please discuss in #156